src/proxy-cssom.js

```diff
@@ -350,10 +359,10 @@ export function installCSSOM() {
     });
   }
 
-  for (let type in cssOMTypes) {
+  for (let [type, value] of Object.entries({'CSSMathValue': CSSMathValue, 'CSSNumericValue': CSSNumericValue, ...cssOMTypes})) {
```

Comment:
I think it would be cleaner to add these to cssOMTypes, e.g.
```js
const cssOMTypes = {
  'CSSMathValue': CSSMathValue,
  'CSSNumericValue': CSSNumericValue,
  /* existing code ... */
};

Reply:
I can't directly, because other classes extend those two classes. Could push them on the object though, right after declaring them but the net result is the same as the code I have right now.

Reply:
No, it can't find the base class because it is not registered yet when declaring the new class that extends it.

Reply:
Does the commit I pushed not work?

Reply:
Just saw that. That should work indeed. Build/Test is running.